about/contact.md

```diff
@@ -3,9 +3,30 @@
 Are you interested in learning more about OceanHackWeek, helping out with the next event, or learning about the next event and how to prepare a strong application? You can:
 
 - Look over this web site, both general pages and [past events](pasthackweeks)
+- Sign up for our email list below
 - Write to us at [info@oceanhackweek.org](mailto:info@oceanhackweek.org).
 - Post a comment or question on the [OceanHackWeek GitHub Discussions](https://github.com/orgs/oceanhackweek/discussions) to engage a larger fraction of the OceanHackWeek community
 - Find us on Twitter at [https://twitter.com/oceanhackweek](https://twitter.com/oceanhackweek)
 - Look up our [FAQ for Applicants](../ohw22/applicants.md#faqs). Most of the FAQ entries are relevant for most years, but bear in mind that we update the FAQ every year to reflect the event being planned.
 
+## Mailing list
+
+To generally stay in touch, you can sign up for our email list. This is also the best way to know when we are planning an event and open applications.
```

Comment:
This isn't a list that subscribers can post to, right? It's a one-way broadcast/announcements list.

```suggestion
Sign up for our email announcements list to stay in touch. This is also the best way to know when we are planning an event and open applications.
```

Reply:
Ya. There isn't a distinct address for it.